notebooks/Lab8.ipynb

```diff
@@ -69,36 +67,56 @@
     "editable": true
    },
    "source": [
-    "As a filtering exercise, we will count the number of distinct patients in the dataset. You can mock this filtering functionality in future questions."
+    "## Filtering exercise\n",
+    "The next few cells will show you how to filter rows based on a condition. Let us select the patients with `Chromosome` as `21`. Let us name this new table as `chromosome_21`. You can mock this filtering functionality in future questions. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "chromosome_21 = mutations.where(\"Chromosome\", \"21\")\n",
+    "chromosome_21"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The next cell creates a table that gives us the distinct patients (notice that in the previous table, the same patient has been repeated) multiple times. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
     "deletable": true,
     "editable": true
    },
    "outputs": [],
    "source": [
-    "# get individual patients\n",
-    "patients = mutations['patient'].drop_duplicates()\n",
-    "patients\n"
+    "# Dropping duplicates from the previous table. These are the individual patients with Chromosome = 21\n",
+    "patients_21 = chromosome_21.select('patient').group('patient').drop(1)\n",
+    "patients_21"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
+    "collapsed": true,
     "deletable": true,
     "editable": true
    },
    "outputs": [],
    "source": [
-    "# count the number of distinct patients\n",
-    "patients.count()"
+    "# counts the number of distinct patients with chromosome 21\n",
+    "patients_21.num_rows"
```

Comment:
This gives the student the skewed perception that there are less patients then actual present in the study. Can you remove this "where" example?